### DISCUSSION TITLE
Help to setup the Dev environment

### DISCUSSION THREAD

**Discussion by naveenrpendli** [ConsenSys/quorum#1329] [Q&A] [answered]
Hello Team,

I'm trying to setup the environment by cloning the repo and running into the issues of the vulnerabilities, etc. Can someone share the stack version details to have a successful run of this repo. Any help is greatly appreciated. Please provide me :

Operating Systems (I have Ubumtu, MAC OS, Windows (both server and desktop)
Version of Python, Node, etc.

Also pitch in any advices or suggestion that you think will be helpful to me have a successful run of this repo.

Thanks
Naveen

**Comment by baptiste-b-pegasys** [comment]
Hello,

For GoQuorum, we are using either VSCode or IntelliJ with GoLang plugins.
You have to setup GoLang 1.16.x https://go.dev/dl/#go1.16.14

`go mod tidy` is useful for downloading all modules.

**Comment by naveenrpendli** [comment]
Please help me to resolve this error

**Reply by antonydenyer** [reply]
❯ go version
go version go1.17 linux/amd64

to make the project run `make`

**Comment by Julian332** [comment, marked as answer]
You need golang(15 or higher ), gcc and linux (i using ububtu 18 ),

**Reply by naveenrpendli** [reply]
> You need golang(15 or higher ), gcc and linux (i using ububtu 18 ),

Do you mind helping me in setting this up once, I can work with you on a Zoom call. I have tried on different operating systems and got stuck with this.

**Reply by Julian332** [reply]
You can show the errors, including text and screenshots
With that, I can check it out for you.
There is a big time difference between us. I'm afraid I can't help you with room .

**Reply by naveenrpendli** [reply]
I can work just once in your time zone to have a good build. I do not have issues working one time in your time zone. This is important for me in understanding few things and I have spent enough time in trying the things out, I don't want to waste more by delaying my things up.

My time zone is US : CST

Thanks
Naveen
